.github/workflows/CI.yml

```
@@ -33,7 +33,7 @@ jobs:
         with:
           repository: 'opensearch-project/OpenSearch'
           path: OpenSearch
-          ref: '1.0'
+          ref: 'main'
       - name: Build OpenSearch
         working-directory: ./OpenSearch
         run: ./gradlew publishToMavenLocal -Dbuild.snapshot=false
```

We are consuming OpenSearch 1.1.0-SNAPSHOT but publishing OpenSearch 1.1.0 to maven. 
Can you move it to publish the snapshot, this would unblock the builds in CI.

```suggestion
        run: ./gradlew publishToMavenLocal
```

---

Missed this, I have updated it. Thank you :)